policyengine_canada/variables/gov/provinces/sk/tax/income/credits/spouse_or_common_law_partner_amount/sk_spouse_or_common_law_partner_credit.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_canada.model_api import *
+
+
+class sk_spouse_or_common_law_partner_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Saskatchewan spouse or common law partner credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.SK
+
+    def formula(household, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.sk.tax.income.credits.spouse_or_common_law_partner_amount
+        person = household.members
+        spouse_income = person("spouse_income", period)
+        live_with_spouse = household("cohabitating_spouses", period) & person(
```

Comment:
this wont work as there are two different entities, we need to make a formula in cohabitating spouses that implies that we have a head and a spouse